database/nakadi/10_data/04_tables/05_timeline.sql

```diff
@@ -8,6 +8,7 @@ CREATE TABLE zn_data.timeline (
   tl_switched_at     TIMESTAMP WITH TIME ZONE DEFAULT NULL,
   tl_cleanup_at      TIMESTAMP WITH TIME ZONE DEFAULT NULL,
   tl_latest_position JSONB                    DEFAULT NULL,
+  tl_deleted         BOOLEAN                  DEFAULT FALSE,
```

Comment:
Ok, this is an interesting point. But I think that it should be fixed from other side. When the  storage is deleted it should also delete all timelines of this storage. And it should also check if at least one timeline is still used - then it should forbid deletion of storage.
I will add these changes to storage deletion.